rewrite-maven/src/main/java/org/openrewrite/maven/tree/MavenMetadata.java

```diff
@@ -62,10 +55,15 @@ public static class Versioning {
         @Nullable
         Snapshot snapshot;
 
+        @Nullable
+        ZonedDateTime lastUpdated;
```

Comment:
Adding a nullable field should be fine right? Since I also do not see MavenMetadata persisted anywhere.